recovery/redo/mlog.go

```diff
@@ -23,14 +23,15 @@ import (
 	"github.com/zbdba/db-recovery/recovery/utils"
 )
 
-func (parse *Parse) MLOG_N_BYTES(data []byte, pos *uint64, MyType uint64) error {
+// MLOG_N_BYTES ...
+func (parse *Parse) mlogNBytes(data []byte, pos *uint64, myType uint64) error {
```

Comment:
I wanted to keep the naming consistent with MySQL, but it was fine to use Golang style with appropriate comments.